.github/workflows/preview-deploy.yml

```diff
@@ -65,15 +65,16 @@ jobs:
         run: |
           gh run download ${{ github.event.workflow_run.id }} \
             --repo "${GITHUB_REPOSITORY}" \
-            --name docs
+            --name docs \
+            --dir html
 
       - uses: elastic/docs-builder/.github/actions/aws-auth@main
 
       - name: Upload to S3
         env:
           PR_NUMBER: ${{ steps.pull_request.outputs.number }}
         run: |
-          aws s3 sync . "s3://elastic-docs-v3-website-preview/${GITHUB_REPOSITORY}/pull/${PR_NUMBER}" --delete --exclude "pull_request.json"
+          aws s3 sync ./html "s3://elastic-docs-v3-website-preview/${GITHUB_REPOSITORY}/pull/${PR_NUMBER}" --delete
```

Comment:
This works today because we publish `links.json` under `html`. 

`.artifacts/docs/links.json` would have been better than `.artifacts/docs/links.json`